lib/utils.js

```diff
@@ -19,9 +19,15 @@ const CD_BASE_DIR = path.resolve(__dirname, '..', '..', 'chromedriver');
 const CD_CDN = process.env.npm_config_chromedriver_cdnurl
   || process.env.CHROMEDRIVER_CDNURL
```

Comment:
Not for this PR. In China, sometimes `https` does not work, but `http` works.  (I learned it recently)
So, this environment variable, `CHROMEDRIVER_CDNURL`, should be here to allow users to configure for their env.

Reply:
yeah, the great firewall of China